certbot/certbot/_internal/main.py

```diff
@@ -1770,12 +1770,31 @@ def reconfigure(config: configuration.NamespaceConfig,
     if not renewal_candidate:
         raise errors.ConfigurationError("Could not load certificate. See logs for errors.")
 
+    renewalparams = orig_renewal_conf['renewalparams']
+    # If server was set but hasn't changed and no account is loaded,
+    # load the old account because reconstitute won't have
+    if lineage_config.set_by_user('server') and lineage_config.server == renewalparams['server']\
+        and lineage_config.account is None:
+        lineage_config.account = renewalparams['account']
+    for param in ('account', 'server',):
+        if getattr(lineage_config, param) != renewalparams.get(param):
+            msg = ("Using reconfigure to change the ACME account or server is not supported. "
+                   "If you would like to do so, use certonly with the --force-renewal flag instead "
+                   "of reconfigure. Note that doing so will count against any rate limits. For "
+                   "more information on this method, see "
+                   "https://certbot.org/certonly-reconfiguration")
```

Comment:
I suspect you're well aware of this but as a reminder for me/us, as of writing this, this link does not work.

Reply:
Yes but thank you for the reminder, I figured I'd do it after approval before merging, so now.